src/app/screens/Settings.tsx

```diff
@@ -133,8 +133,18 @@ function Settings() {
               label={t("camera_access.allow")}
               onClick={async () => {
                 try {
-                  await Html5Qrcode.getCameras();
-                  setCameraPermissionsGranted(true);
+                  const devices = await QrScanner.listCameras();
+                  const video =
+                    document.querySelector<HTMLVideoElement>("#reader");
```

Comment:
We control the name of the ID here, don't we? Can we change it to a more descriptive name like `#qr-code-reader`?